README.md

```diff
@@ -73,6 +74,7 @@ Flags:
       --key string                     identify secure client using this TLS key file
       --move-leader                    whether to move the leadership before performing defragmentation on the leader
       --password string                password for authentication (if this option is used, --user option shouldn't include password)
+      --skip-healthcheck-cluster-endpoints   skip cluster endpoint discovery during health check and only check the endpoints provided via --endpoints
```

Comment:
This isn't well formatted. Pls execute `./etcd-defrag -h` and paste & replace the whole output